Source/FolioReaderConfig.swift

```diff
@@ -37,12 +39,54 @@ public enum FolioReaderScrollDirection: Int {
     }
 }
 
+// MARK: - ClassBasedOnCLickListener
+
+/**
+A `ClassBasedOnCLickListener` takes a closure which is performed if a given html `class` is clicked. The closure will reveice the content of the specified parameter.
+
+Eg. A ClassBasedOnCLickListener with the className "quote" and parameterName "id" with the given epub html content "<section class="quote" id="12345">" would call the given closure on a click on this section with the String "12345" as parameter.
+
+*/
+public struct ClassBasedOnCLickListener {
```

Comment:
Ok, I will fix it to `ClassBasedOnClickListener`. I'm don't have a better naming yet as the current one describes the function really good. To just use `OnClickListener` or sth. similiar wouln't be clear about it's function and usage.

What name do you have in mind?